src/Raven.Studio/wwwroot/App/views/database/indexes/dumpDialog.html

```diff
@@ -0,0 +1,42 @@
+<div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+        <div class="modal-header word-break">
+            <button type="button" class="close" data-bind="click: close" aria-hidden="true">
+                <i class="icon-cancel"></i>
+            </button>
+            <h3 class="modal-title">Dump Index</h3>
+            <h4 data-bind="text: indexName"></h4>
```

Comment:
I wonder if want to put index name label here?

Reply:
not sure if we really need that - but this is how it can look w/ label:

![image](https://user-images.githubusercontent.com/17376974/143933621-a005dfcd-270f-439c-9e39-49364f6ca598.png)

Reply:
refactored to:

![image](https://user-images.githubusercontent.com/17376974/144010947-4bcaca96-834a-4f2c-ae34-9ab271e8c481.png)